src/st/compatibility/class-wpml-page-builders-register-strings.php

```diff
@@ -53,10 +53,13 @@ public function register_strings( WP_Post $post, array $package ) {
 		$data = get_post_meta( $post->ID, $this->data_settings->get_meta_field(), false );
 
 		if ( $data ) {
-			$this->register_strings_for_modules(
-				$this->data_settings->convert_data_to_array( $data ),
-				$package
-			);
+			$converted = $this->data_settings->convert_data_to_array( $data );
+			if ( is_array( $converted ) ) {
+				$this->register_strings_for_modules(
+					$converted,
+					$package
+				);
+			}
```

Comment:
Note: This code is covered by tests in the Elementor class that extends this class.